ykrt/src/compile/jitc_yk/trace_builder.rs

```diff
@@ -462,14 +467,21 @@ impl<'a> TraceBuilder<'a> {
     /// Entry point for building an IR trace.
     ///
     /// Consumes the trace builder, returning a JIT module.
+    ///
+    /// # Panics
+    ///
+    /// If `ta_iter` produces no elements.
```

Comment:
Unrelated to this PR, but I'm not sure what `ta` means here.

Reply:
:man_shrugging:

Reply:
[It means `trace_iter` FWIW.]

Reply:
Should it be `ti`? Let's move on, it's not worth worrying about.

Reply:
Oops! "TraceAction_iter" I meant.

Reply:
Ah, that makes more sense!